docs/project-structure/project-template.md

```diff
@@ -0,0 +1,149 @@
+# Project Template
+
+The Genezio project template showcases how you can structure large projects with Genezio. This template can be used as a starting point for your project or as a way to learn how to organize your project according to the best practices. For this project, we created a dummy todo application to showcase all the functionalities of Genezio.
+
+The whole project can be found in this [GitHub repository](https://github.com/Genez-io/ultimate-project-template.git).
+
+## Prerequisites
+
+- [NodeJS](https://nodejs.org/en/download/) >= 18.2
+- [MongoDB URL](https://www.mongodb.com/resources/products/fundamentals/mongodb-cluster-setup)
+- [PostgreSQL URL](/docs/features/databases/#how-to-create-a-database)
```

Comment:
are both of these required at the same time? if not, it would be best to mention that the project supports both, and the user can choose their preferred db.

Reply:
they are required because we have two services, one for postgres and one for mongo